templates/repo/issue/fields/textarea.tmpl

```diff
@@ -1,6 +1,22 @@
-<div class="field">
+<div class="field markdown-textarea">
 	{{template "repo/issue/fields/header" .}}
 	{{/* FIXME: preview markdown result */}}
-	{{/* FIXME: required validation for markdown editor */}}
-	<textarea name="form-field-{{.item.ID}}" placeholder="{{.item.Attributes.placeholder}}" {{if and .item.Validations.required .item.Attributes.render}}required{{end}}>{{.item.Attributes.value}}</textarea>
+	<textarea class="textarea" name="form-field-{{.item.ID}}" placeholder="{{.item.Attributes.placeholder}}" {{if and .item.Validations.required .item.Attributes.render}}required{{end}}>{{.item.Attributes.value}}</textarea>
+
+	{{$ := .Context.Data}}
```

Comment:
It doesn't look right.

1. The Golang's standard should be broken, aka, the `$` should not be changed.
2. The `.Context.Data` seems not good and fragile.

Reply:
Sorry for my poor knowladge.
I tried to fix this in https://github.com/go-gitea/gitea/pull/24400/commits/ff83b3b1cdbe56f68fece8a786a22a27f761dabb, dose it look good to you?